### ISSUE TITLE
Convert to wide format not working on 3.9

### ISSUE THREAD

**Issue by alexgonca** [strohne/Facepager#75]
I tried to export my nodes using the "convert to wide format" option.
It exports but when you open the file the data is not in the "wide format".
I am running Facepager on Windows 10.

BTW thank you so much for the great work!

**Comment by strohne** [comment]
Thank you for the bug report. Is there a message in the status log?

It should work if you set the export mode to "All nodes". I will fix this in the next release.

**Comment by alexgonca** [comment]
Sorry for my delay to answer! My students were using Facepager 3.7 and I had to uninstall 3.9 to be able to help them.
Last night, I reinstalled Facepager 3.9... and now it is working. I have just exported a CSV as 
"wide format" and it looks fine:
https://docs.google.com/spreadsheets/d/1S9d39ov8OdF9Ny99DPEW7hlyGtf7Uj7VqcpGosLU0r4/edit?usp=sharing
Have you updated the build? If not, I probably committed some mistake when I was trying to export as "wide format". If that was the case, I am sorry!
